AzureCalling/app/src/main/java/com/azure/samples/communication/calling/activities/CallActivity.java

```diff
@@ -375,12 +387,28 @@ private void openShareDialogue() {
     }
 
     private void setLayoutComponentState(
-            final boolean isMicrophoneMuted, final boolean isCameraOn, final boolean isCallHangUpOverLaid) {
+            final boolean isMicrophoneMuted, final boolean isCameraOn,
+            final boolean isCallHangUpOverLaid) {
         audioImageButton.setSelected(!isMicrophoneMuted);
         videoImageButton.setSelected(isCameraOn);
         callHangupOverlay.setVisibility(isCallHangUpOverLaid ? View.VISIBLE : View.INVISIBLE);
     }
 
+    private void setAudioDeviceListVisible() {
+        if (!isPopupWindowVisible) {
+            final AudioSessionManager audioSessionManager
+                    = ((AzureCalling) getApplicationContext()).getAudioSessionManager();
+            audioDeviceSelectionPopupWindow = new AudioDeviceSelectionPopupWindow(this, audioSessionManager);
+        } else {
+            audioDeviceSelectionPopupWindow.update();
+        }
```

Comment:
We can use the same code as from SetupActivity (ie. don't worry about the flag)
``` java
private void setAudioDeviceListVisible() {
        final AudioSessionManager audioSessionManager
                = ((AzureCalling) getApplicationContext()).getAudioSessionManager();
        audioDeviceSelectionPopupWindow = new AudioDeviceSelectionPopupWindow(this, audioSessionManager);
    }
```

Reply:
Yea, since the popup window can handle the screen rotation itself, we won't need to handle it anymore. Updated now.